modules/tensor_mechanics/src/materials/GeneralizedKelvinVoigtBase.C

```diff
@@ -20,31 +20,38 @@ validParams<GeneralizedKelvinVoigtBase>()
 }
 
 GeneralizedKelvinVoigtBase::GeneralizedKelvinVoigtBase(const InputParameters & parameters)
-  : LinearViscoelasticityBase(parameters)
+  : LinearViscoelasticityBase(parameters),
+    _first_elasticity_tensor_old(
+        getMaterialPropertyOld<RankFourTensor>(_base_name + "first_elasticity_tensor")),
+    _first_elasticity_tensor_inv_old(
+        getMaterialPropertyOld<RankFourTensor>(_base_name + "first_elasticity_tensor_inv")),
+    _springs_elasticity_tensors_inv_old(getMaterialPropertyOld<std::vector<RankFourTensor>>(
+        _base_name + "springs_elasticity_tensors_inv"))
 {
-  _need_viscoelastic_properties_inverse = true;
+  //  _need_viscoelastic_properties_inverse = true;
```

Comment:
It looks like this gets set in the parameters, so there's no need to set it (which I assume is why you commented it out).  Delete this commented out line of code if that's the case.

Reply:
Yes, it is set in the parameters (I commented it to check whether it was set correctly). I forgot to remove it...